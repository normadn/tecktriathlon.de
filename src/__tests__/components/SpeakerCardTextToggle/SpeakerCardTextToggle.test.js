import React from "react";
import SpeakerCardTextToggle from "../../../components/SpeakerCardTextToggle";
import { shallow } from "enzyme";

describe("SpeakerCardTextToggle", () => {
  it("renders correctly", () => {
    expect(
      shallow(
        <SpeakerCardTextToggle
          focusedText="focused!"
          defaultText="default!"
          focused={true}
          onClick={() => {}}
        />
      )
    ).toMatchSnapshot();
  });
});
