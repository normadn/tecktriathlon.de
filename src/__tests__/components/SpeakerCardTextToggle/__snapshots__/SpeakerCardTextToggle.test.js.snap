// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpeakerCardTextToggle renders correctly 1`] = `
<div>
  <button
    css={
      Object {
        ":hover": Object {
          "color": "#00205b",
        },
        "background": "none",
        "border": "none",
        "color": "#a5a5a5",
        "cursor": "default",
        "fontSize": "16px",
        "fontWeight": "bold",
        "textTransform": "uppercase",
        "transition": "1s",
      }
    }
    onClick={[Function]}
  >
    focused!
  </button>
</div>
`;
